### ISSUE TITLE
Rendering area is always 640x480 on Wine

### ISSUE THREAD

**Issue by zerkerX** [dpjudas/UT99VulkanDrv#1]
Hello. First I have to say: this is great! Thanks for doing this.

Now, as you gathered from the subject, I am running in a somewhat non-standard configuration, but Linux and Wine usually have very good Vulkan support, so I'm confused as to what is happening here. Probably some ambiguous part of the standard where one implementation does one thing, and another does something else :)

When I start the game, I see a 640x480 rendering area in the upper-left of the screen, which renders the top-left of the game view.

![Screenshot_20211114_113725](https://user-images.githubusercontent.com/45739550/141696142-cecc1561-d6b2-48ce-97d7-f420ac9c5d7b.png)

If I use ``setres 640x480``, then I can see the full view area.

![Screenshot_20211114_113655](https://user-images.githubusercontent.com/45739550/141696144-dfafb5d5-46c1-4026-84ae-4490a4109cb3.png)

I can also set the Vulkan renderer as the windowed renderer, which provides me with a sizable window, but again only renders 640x480 pixels. The rest is filled with noise.

![Screenshot_20211114_114453](https://user-images.githubusercontent.com/45739550/141696149-ebff4eec-9801-4918-a4e6-8409ff38207c.png)

If there is any information I can gather for you that will help, please let me know. I have tested with both Wine 5.0.3 and Wine 6.0. Mesa drivers are version 21.2.5 with Kernel 5.14. Some other general system info is in the screenshot if relevant.

**Comment by dpjudas** [comment]
Are you using UT patch 436 or 469 for this? And also, does the OpenGL driver do the same?

**Comment by zerkerX** [comment]
436. And the OpenGL driver works flawlessly. I guess I can try the 469 patch.

**Comment by dpjudas** [comment]
Hmm, I'm not sure what is causing it. The resizing works fine on Windows and there is no 640x480 hardcoding in the vulkan driver itself. Hopefully 469 will fix it. :)

**Comment by zerkerX** [comment]
Unfortunately, 469 behaves the same.

**Comment by zerkerX** [comment]
That said, having found 469, I realize that gets the Linux-native version of UT functional again. So I probably won't be using this in Wine going forward ;)

So I guess this is probably an irrelevant bug unless you or someone else starts building this driver on Linux and it behaves the same with the native version of the game.

**Comment by dpjudas** [comment]
Yes, was about to ask you why you weren't using the native version. :) Tbh I would rather try to compile the driver for Linux than fix the wine issue. The OpenGL driver that comes with 469 should probably be just fine though.
